### ISSUE TITLE
fix tests for pytest 4+

### ISSUE THREAD

**Issue by sandrotosi** [pauldmccarthy/indexed_gzip#21]
Hello,
i just applied this patch to make tests work on newer pytest, 4+
```
--- a/conftest.py
+++ b/conftest.py
@@ -80,17 +80,16 @@ def seed(request):
 
 
 @pytest.fixture
-def testfile(request):
+def testfile(request, nelems, concat):
 
     filename = request.config.getoption('--testfile')
-    _nelems  = nelems(request)
-    _concat  = concat(request)
 
     if filename is None:
         filename = op.join(os.getcwd(),
-                           'ctest_zran_{}_{}.gz'.format(_nelems, _concat))
+                           'ctest_zran_{}_{}.gz'.format(nelems, concat))
 
     if not op.exists(filename):
-        gen_test_data(filename, _nelems, _concat)
+        gen_test_data(filename, nelems, concat)
 
     return filename
+
```

**Comment by sandrotosi** [comment]
https://docs.pytest.org/en/latest/deprecations.html#calling-fixtures-directly

**Comment by pauldmccarthy** [comment]
Thanks! But it looks like you might have been working against an older version - this has been fixed in `indexed_gzip` since version 0.8.8 ..

**Comment by sandrotosi** [comment]
ouch, i even looked but i did not notice. ok sorry for the noise
